README.md

```diff
@@ -1 +1,84 @@
-# node-process
\ No newline at end of file
+# Node Process Object
+
+## Overview
+
+This lesson will cover the `process` global object in Node to get system information and manipulate with the Node instance. 
+
+## Objectives
+
+1. Describe the process objects
+1. Describe about process information such as versions, pid, cwd, features, uptime, memoryUsage
+1. Describe how to exit the process
+1. Describe environment variables
+2. Describe standard input and output
+
```

Comment:
Need have to an intro which presents a problem is which it is useful to know a bunch of system information - as well as a definition of what `process` is